source/administration/object-management/object-delete.rst

```diff
@@ -0,0 +1,176 @@
+
+Delete behavior on any single peer follows the same processes as any MinIO deployment.
+
+Bucket Replication
+~~~~~~~~~~~~~~~~~~
+
+With :ref:`bucket replication <minio-bucket-replication>`, MinIO supports replicating delete operations between a source bucket and a configured remote bucket.
+MinIO synchronizes deleting specific object versions *and* new  :s3-docs:`delete markers <delete-marker-replication.html>`. 
+Delete operation replication uses the same :ref:`replication process <minio-replication-process>` as all other replication operations. 
+
+MinIO requires *explicitly enabling* versioned deletes and delete marker replication. 
+Use the :mc-cmd:`mc replicate add --replicate` field to specify either ``delete`` and ``delete-marker`` or both to enable versioned deletes and delete marker replication, respectively. 
+To enable both, specify both strings using a comma separator ``delete,delete-marker``.
+
+For delete marker replication, MinIO begins the replication process after a delete operation creates the delete marker. 
+MinIO uses the ``X-Minio-Replication-DeleteMarker-Status`` metadata field for tracking  delete marker replication status. 
+In :ref:`active-active <minio-bucket-replication-serverside-twoway>` replication configurations, MinIO may produce duplicate delete markers if both clusters concurrently create a delete marker for an object *or* if one or both clusters were down before the replication event synchronized.
+
+For replicating the deletion of a specific object version, MinIO marks the object version as ``PENDING`` until replication completes. 
+Once the remote target deletes that object version, MinIO deletes the object on the source.
+While this process ensures near-synchronized version deletion, it may result in listing operations returning the object version after the initial delete operation. 
+MinIO uses the ``X-Minio-Replication-Delete-Status`` for tracking delete version replication status.
+
+MinIO only replicates explicit client-driven delete operations. 
+MinIO does *not* replicate objects deleted from the application of :ref:`lifecycle management expiration rules <minio-lifecycle-management-expiration>`. 
+For :ref:`active-active <minio-bucket-replication-serverside-twoway>` configurations, set the same expiration rules on *all* of of the replication buckets to ensure consistent application of object expiration.
+
+.. admonition:: MinIO Trims Empty Object Prefixes on Source and Remote Bucket
+   :class: note, dropdown
+
+   If a delete operation removes the last object in a bucket prefix, MinIO recursively removes each empty part of the prefix up to the bucket root.
+   MinIO only applies the recursive removal to prefixes created *implicitly* as part of object write operations - that is, the prefix was not created using an explicit directory creation command such as :mc:`mc mb`.
```

Comment:
An idea to break this up (with a hint that it could have been created by something besides mc).
```suggestion
   MinIO only applies the recursive removal to prefixes created *implicitly* as part of object write operations.
Prefixes created using an explicit directory creation operation, such as :mc:`mc mb`, are not recursively removed.
```

Reply:
Brain couldn't fix that sentence. Glad yours could.